recipes/arb-bio/6.1.rev16223/abs_path_fix.patch

```diff
@@ -0,0 +1,753 @@
+Index: BINDINGS/PERL/Makefile
+===================================================================
+--- BINDINGS/PERL/Makefile	(revision 16223)
++++ BINDINGS/PERL/Makefile	(working copy)
```

Comment:
Hmm. In theory I could script a replacement of `#!/usr/bin/perl` to the current absolute perl path. Conda should then pick that up and change it as it does with all the python script paths when it installs the package.